docs/test/using-microsoft-visualstudio-testtools-unittesting-members-in-unit-tests.md

```diff
@@ -353,6 +351,34 @@ The following attributes and the values assigned to them appear in the Visual St
 
 - <xref:Microsoft.VisualStudio.TestTools.UnitTesting.WorkItemAttribute>
 
+### DeploymentItemAttribute
+
+The <xref:Microsoft.VisualStudio.TestTools.UnitTesting.DeploymentItemAttribute> for copying files or folders specified as deployment items to the deployment directory. Deployment directory is where all the deployment items are present along with TestSource dll.(starting from MSTest v2)
+
+It can be used either on test classes (classes marked with `TestClass` attribute) or on test methods (methods marked with `TestMethod` attribute).
+
+Users can have multiple instances of the attribute to specify more than one item.
+
+And here you can see its [constructors](<xref:Microsoft.VisualStudio.TestTools.UnitTesting.DeploymentItemAttribute>).
+
+**Example**
+
+```csharp
+// without adding a custom output path the copied files will be in TestResults folder inside the project folder
```

Comment:
I would remove this comment here and add it in the explanation above (see previous comment).